gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -271,7 +281,10 @@ private boolean isLocationLockEnabled() {
 
   public void onCameraMove(CameraPosition newCameraPosition) {
     Timber.d("Setting position to %s", newCameraPosition.toString());
-    this.cameraPosition.setValue(newCameraPosition);
+    cameraPosition.setValue(newCameraPosition);
+    Loadable.getValue(projectLoadingState)
+        .ifPresent(
```

Comment:
Is this equivalent to "getLastActiveProjectId()" in ProjectRepository? If yes, then we should simplify it by replacing with that.

Reply:
Sorry didn't have a chance to get back to this yet.  It's not equivalent, since `getLastActiveProjectId()` gives us the id of the last project to successfully load, not necessarily in the current session. This could differ if the app is restarted and the project is still reloading, or if the user just selected a new project that hasn't finished loading yet.